firebase-ai-logic/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "vite-starter",
```

nit: change the project demo from the default.

```suggestion
  "name": "firebase-ai-logic",
```